dockerfiles/build.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/sh
 
 set -ex
-
+if [[ -z ${DOCKER} ]]; then
+    command -v docker >/dev/null 2>&1 && DOCKER=docker
+fi
+if [[ -z ${DOCKER} ]]; then
+    command -v podman >/dev/null 2>&1 && DOCKER=podman
+fi
```

Comment:
`[[` won't work if `/bin/sh` is not symlinked to bash. So either adjust the shebang or don't use `[[`.

Reply:
Somewhere back in my brain there rings a bell. It's now `dash` compatible.